shape/Circle.java

```diff
@@ -1,15 +1,20 @@
 package shape;
 
+import exceptions.NegativeLengthException;
+
 public class Circle implements Shape {
-	private final double radius;
+	protected double radius;
 	public Circle(double radius){
 		this.radius = radius ;
+		if (radius < 0)
+			throw new NegativeLengthException(" required Radius > 0 "); 

```

Comment:
always have scope ( brackets ), i.e.:

```suggestion
		if (radius < 0) {
			throw new NegativeLengthException(" required Radius > 0 "); 
		}

```